cli/src/commands/env.ts

```diff
@@ -100,11 +118,15 @@ export default class Env extends Command {
     }
   }
 
-  protected async list(appId: string) {
-    const { variables } = await this.lycanClient.getEnv(appId, null);
-    // Output in sorted order (of variable names).
-    const sorted = variables.map(vvToString).sort();
-    this.log(sorted.join('\n'));
+  protected async list(appId: string, includeValues: boolean) {
+    try {
+      const { variables } = await this.lycanClient.getEnv(appId, null);
+      // Output in sorted order (of variable names).
+      const sorted = (includeValues ? variables : eraseValues(variables)).map(vvToString).sort();
```

Comment:
`sort` without a comparator function is suspicious, although a `=` separator guarantees that we are sorting by variable names and not the whole value.

Reply:
Indeed, the output string itself is sorted.